spec/services/map_activity_service.rb

```diff
@@ -0,0 +1,286 @@
+require 'rails_helper'
+
+RSpec.describe MapActivityService do
+  let(:map) { create(:map, created_at: 1.week.ago) }
+  let(:other_user) { create(:user) }
+  let(:email_user) { create(:user) }
+
+  it 'includes nothing if nothing happened' do
+    response = MapActivityService.summarize_data(map, email_user)
+    expect(response[:stats]).to eq({})
```

Comment:
i'd argue we should be testing the whole hash to be equal to { stats: {} }

We could even make that a constant to check against during these tests

It just communicates more info